### DISCUSSION TITLE
Option to set dark/light apperance without matching the system apperance

### DISCUSSION THREAD

**Discussion by jhoward15** [lawand-dot-io/taskbar#20] [Feature requests]
I'd like the option to be able to choose the dark theme for the taskbar while I run my system in light mode instead of always matching the system theme.

**Comment by lawand-dot-io** [comment]
Setting e.g. to dark mode when the system is in light mode shouldn't be too hard to implement, so I'll consider implementing it at some point. Supporting different colors could be tricky because in some configurations the window title text might not be readable. I'll consider it too. Cheers :)

**Reply by jhoward15** [reply]
I think just being able to pick light/dark should be an 80% solution for those picky like me

**Comment by lawand-dot-io** [comment]
Split the feature request of supporting custom colors into https://github.com/lawand-dot-io/taskbar/discussions/30

**Comment by lawand-dot-io** [comment]
This feature has been implemented in Taskbar version 1.2.1 which is now released. Closing the ticket.
@jhoward15
